src/main/resources/openapiv2.json

```diff
@@ -671,75 +802,130 @@
                     "items": {
-                        "type": "string"
+                        "$ref": "#/definitions/Offset"
                     }
                 }
             },
-            "example": "{\n    \"topics\": [\n        \"topic1\",\n        \"topic2\"\n    ]\n}"
+            "example": {
+                "offsets": [
+                    {
+                        "topic": "topic",
+                        "partition": 0,
+                        "offset": 15
+                    },
+                    {
+                        "topic": "topic",
+                        "partition": 1,
+                        "offset": 42
+                    }
+                ]
+            }
         },
-        "Records": {
-            "title": "Records",
+        "OffsetsWithoutTopic": {
+            "title": "Offsets",
             "type": "object",
             "properties": {
-                "records": {
+                "offsets": {
                     "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/Record"
+                    "items": {}
+                }
+            },
+            "example": {
+                "offsets": [
+                    {
+                        "partition": 2,
+                        "offset": 0
+                    },
+                    {
+                        "partition": 1,
+                        "offset": 1
+                    },
+                    {
+                        "partition": 2,
+                        "offset": 2
                     }
+                ]
+            }
+        },
+        "Partition": {
+            "title": "Partition",
+            "type": "object",
+            "properties": {
+                "partition": {
+                    "format": "int32",
+                    "type": "integer"
+                },
+                "leader": {
+                    "format": "int32",
+                    "type": "integer"
+                },
+                "replicas": {
+                    "type": "object"
                 }
             },
-            "example": "{\n    \"records\": [\n        {\n            \"key\": \"1abcd\",\n            \"value\": \"msg1\"\n        },\n        {\n            \"value\": \"msg2\",\n            \"partition\": 1\n        },\n        {\n            \"value\": \"msg3\"\n        }\n    ]\n}"
+            "example": {
+                "partition": 1,
+                "leader": 1,
+                "replicas": [
+                    {
+                        "broker": 1,
+                        "leader": true,
+                        "in_sync": true
+                    },
+                    {
+                        "broker": 2,
+                        "leader": false,
+                        "in_sync": false
+                    }
+                ]
+            }
         },
-        "Partitions": {
+        "PartitionsForPositions": {
```

Comment:
it's not just for positions endpoint but used on seek as well right? Let's call it just `Partitions` if the previous one is really not used in our case.